docs/index.rst

```diff
@@ -14,6 +14,13 @@ altered in those files.
    overview
    scripts
 
+.. toctree::
+   :caption: Ocean
+   :maxdepth: 2
+
+   ocean
+   ocean_testcases/index
+
 .. toctree::
```

Comment:
@mark-petersen, I'm not sure I agree that core info should go before machine info, but we can try this out.

Reply:
The core info seems more central, while the machine-specific feels more like an appendix. I was afraid people wouldn't see 'Ocean' if it is so low in the TOC.

Reply:
Okay, that makes sense.  I need to remind myself what the TOC looks like.  We wouldn't want all the ocean configurations listed, I guess, because that would make the TOC too long.  As long as they're not in the top-level TOC, I agree that this organization will work.